### PR TITLE
Truncation of course name 

### PR DESCRIPTION
**Pull request by neeraj-2**
This Pr is for truncating long custom-Courses :
image

image

### CODE REVIEW

File: src/pages/Popup/components/AlarmUp.jsx [hunk 1]
```diff
@@ -51,6 +51,10 @@ class Alarmview extends Component {
     }
     this.setState({ alarms: setalarms });
   }
+  
+  trunc(string,num) { 
+    return string.length > num ? `${ string.slice(0, 18)}..` : string
```
Comment (hirenchalodiya1) [inline src/pages/Popup/components/AlarmUp.jsx:56]: `18` should be `num`

File: src/pages/Popup/components/Alarmview.jsx [hunk 2]
```diff
@@ -61,6 +61,10 @@ class Alarmview extends Component {
       });
     }
   }
+  
+  trunc(string,num) { 
+    return string.length > num ? `${ string.slice(0,22)}..` : string
```
Comment (hirenchalodiya1) [inline src/pages/Popup/components/Alarmview.jsx:66]: `22` should be `num`

File: src/pages/Popup/components/Alarmview.jsx [hunk 3]
```diff
@@ -88,7 +92,7 @@ class Alarmview extends Component {
                 </ListItemIcon>
                 <ListItemText
                   id={alarm.code}
-                  primary={alarm.course == 'custom' ? `${alarm.code}` : `${alarm.course}`}
+                 primary={alarm.course == 'custom' ? `${this.trunc(alarm.code, 22)}` : `${alarm.course}`}
```
Comment (hirenchalodiya1) [inline src/pages/Popup/components/Alarmview.jsx:95]: I think you removed space from the beginning mistakenly

File: src/pages/Popup/components/AlarmUp.jsx [hunk 4]
```diff
@@ -51,6 +51,10 @@ class Alarmview extends Component {
     }
     this.setState({ alarms: setalarms });
   }
+  
+  trunc(string,num) {
```
Comment (hirenchalodiya1) [inline src/pages/Popup/components/AlarmUp.jsx:55]: add a space between `string` and `num` variable as good dev practise

**Review by hirenchalodiya1 — CHANGES_REQUESTED** [review]
needs change in both `trunc` functions

File: src/pages/Popup/components/Alarmview.jsx [hunk 5]
```diff
@@ -61,6 +61,10 @@ class Alarmview extends Component {
       });
     }
   }
+  
+  trunc(string,num) {
```
Comment (hirenchalodiya1) [inline src/pages/Popup/components/Alarmview.jsx:65]: Same changes that you made in other file

**Review by hirenchalodiya1 — CHANGES_REQUESTED** [review]

**Review by hirenchalodiya1 — APPROVED** [review]

File: src/pages/Popup/components/AlarmUp.jsx (hunk 4 above)
Comment (DarshitJain04) [reply src/pages/Popup/components/AlarmUp.jsx]: @neeraj-2 Use prettier as it formats the code nicely

**Pull request merged**